scripts/common.sh

```diff
@@ -103,19 +105,19 @@ EOF
 
 configure_apt_local_repository() {
     local -r repo_name=$1
-    local -r repo_path="$BASE_DIR/packages/debian/$repo_name"
+    local -r repo_path="$BASE_DIR/packages/debian/$OS_VERSION/$repo_name"
     echo "deb [trusted=yes] file://$repo_path bionic $repo_name" \
         > /etc/apt/sources.list.d/"$repo_name".list
 }
 
 get_packages_list() {
-    python -c "
+    python3 -c "
```

Comment:
It cannot works on rhel-7 since python3 may not be installed when we run this command

Reply:
Similarly, I seem to remember RHEL8 could come with no `python` pre-installed (except for the non-system-one installed for `dnf` to function) at all?